api/consensus.go

```diff
@@ -4,28 +4,17 @@ import (
 	"net/http"
 
 	"github.com/NebulousLabs/Sia/build"
-	"github.com/NebulousLabs/Sia/crypto"
 	"github.com/NebulousLabs/Sia/types"
 )
 
-// The ConsensusSetStatus struct contains general information about the
-// consensus set, with tags to have idiomatic json encodings.
-type ConsensusSetStatus struct {
+// The ConsensusGET struct contains general information about the consensus
+// set, with tags to support idiomatic json encodings.
+type ConsensusGET struct {
 	Height       types.BlockHeight `json:"height"`
 	CurrentBlock types.BlockID     `json:"currentblock"`
 	Target       types.Target      `json:"target"`
```

Comment:
It seems acceptable to me to add json tags to all of the structs in the types package